docs/fundamentals/code-analysis/style-rules/language-rules.md

```diff
@@ -118,7 +118,7 @@ The style rules in this section are applicable to C# language only.
 
 ### Implicit and explicit types
 
-The style rules in this section concern the use of the [var](/dotnet/csharp/language-reference/keywords/var) keyword versus an explicit type in a variable declaration. This rule can be applied separately to built-in types, when the type is apparent, and elsewhere.
```

Comment:
I am actually deleting all the sections in this file in https://github.com/dotnet/docs/pull/20892

Reply:
That PR also updates these links to be relative paths.